.github/workflows/build.yml

```
@@ -419,7 +434,7 @@ jobs:
 
   # Catch-all required check for test matrix and coverage
   test-success:
-    needs: [setup, test, coverage]
+    needs: [setup, test, codecov, coverage-threshold]
```

```suggestion
    needs: 
      - setup
      - test
      - codecov
      - coverage-threshold
```

we should use array style for less future changes